site/docs/guides/migration.md

```diff
@@ -76,11 +76,11 @@ Exported Nessie repository, 100 commits into 1 files, 1 named references into 1
 Once the
 export process has completed, the ZIP file will be available at `/tmp/export-<date>.zip`.
 
-Download the `nessie-quarkus-cli-x.y.z-runner.jar` corresponding to the **new** Nessie version, here
-we assume version 0.75.0:
+Download the `nessie-admin-cli-x.y.z-runner.jar` corresponding to the **new** Nessie version, here
```

Comment:
Changes in this file need to go into a separate PR (similar to https://github.com/projectnessie/nessie/pull/8468)

Reply:
#8484